query-service-impl/build.gradle.kts

```diff
@@ -40,10 +45,10 @@ dependencies {
   implementation("org.slf4j:slf4j-api:1.7.32")
   implementation("commons-codec:commons-codec:1.15")
   implementation("org.hypertrace.core.serviceframework:platform-metrics:0.1.33")
-  implementation("com.google.protobuf:protobuf-java-util:3.19.2")
+  implementation("com.google.protobuf:protobuf-java-util:3.20.1")
   implementation("com.google.guava:guava:30.1.1-jre")
   implementation("io.reactivex.rxjava3:rxjava:3.0.11")
-  implementation("com.squareup.okhttp3:okhttp:4.9.1")
+  implementation("com.squareup.okhttp3:okhttp:5.0.0-alpha.6")
```

Comment:
The issue is not fixed till `[4.10.0-RC1](https://mvnrepository.com/artifact/com.squareup.okhttp3/mockwebserver/4.10.0-RC1)` - `https://mvnrepository.com/artifact/com.squareup.okhttp3/mockwebserver`

The issue was for this:

```
Improper Locking [Medium Severity][https://snyk.io/vuln/SNYK-JAVA-ORGJETBRAINSKOTLIN-2628385] in org.jetbrains.kotlin:kotlin-stdlib@1.4.10
```

Reply:
Shall I move to `4.9.3` (as above are alpha releases) and put constraints?

Reply:
alpha releases are not safe to use in production. 
its better we add constraint till we get a stable release

Reply:
> alpha releases are not safe to use in production. its better we add constraint till we get a stable release

I was in the same doubt.

Reply:
Done